common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -242,6 +243,21 @@ def test_showanswer_default(self):
         problem = CapaFactory.create()
         self.assertFalse(problem.answer_available())
 
+    @ddt.data(
+        (requests.exceptions.ReadTimeout, (1, 'Failed to read from the server')),
+        (requests.exceptions.ConnectionError, (1, 'cannot connect to server')),
+    )
+    @ddt.unpack
+    def test_timeout_exceptions(self, exception, result):
+        """
+        Makes sure that platform will raise appropriate exception
```

Comment:
I am not sure the doc string following the [edx python guidelines. ](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html)

>  Please use 79 chars as a limit for docstring lines though, so that the text remains readable.